sdk/keyvault/azure-security-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/package-info.java

```diff
@@ -2,7 +2,161 @@
+ * using the {@link com.azure.security.keyvault.secrets.SecretClient}.</p>
+ *
+ * <!-- src_embed com.azure.security.keyvault.SecretClient.setSecret#string-string -->
+ * <pre>
+ * KeyVaultSecret secret = secretClient.setSecret&#40;&quot;secretName&quot;, &quot;secretValue&quot;&#41;;
+ * System.out.printf&#40;&quot;Secret is created with name %s and value %s%n&quot;, secret.getName&#40;&#41;, secret.getValue&#40;&#41;&#41;;
+ * </pre>
+ * <!-- end com.azure.security.keyvault.SecretClient.setSecret#string-string -->
+ *
+ * <p><strong>Asynchronous Code Sample:</strong></p>
+ * <p>The following code sample demonstrates how to asynchronously create and store a secret in the key vault,
+ * using the {@link com.azure.security.keyvault.secrets.SecretAsyncClient}.</p>
+ *
+ * <!-- src_embed com.azure.keyvault.secrets.SecretClient.setSecret#string-string -->
+ * <pre>
+ * secretAsyncClient.setSecret&#40;&quot;secretName&quot;, &quot;secretValue&quot;&#41;
+ *     .subscribe&#40;secretResponse -&gt;
+ *         System.out.printf&#40;&quot;Secret is created with name %s and value %s%n&quot;,
+ *             secretResponse.getName&#40;&#41;, secretResponse.getValue&#40;&#41;&#41;&#41;;
+ * </pre>
+ * <!-- end com.azure.keyvault.secrets.SecretClient.setSecret#string-string -->
+ *
+ * <hr/>
+ *
+ * <h2>Get a Secret</h2>
+ * The {@link com.azure.security.keyvault.secrets.SecretClient} or
+ * {@link com.azure.security.keyvault.secrets.SecretAsyncClient} can be used to retrieve a secret from the
+ * Azure KeyVault.
+ *
+ * <p><strong>Synchronous Code Sample:</strong></p>
+ * <p>The following code sample demonstrates how to synchronously retrieve a previously stored secret from the Azure
+ * KeyVault, using the {@link com.azure.security.keyvault.secrets.SecretClient}.</p>
+ *
+ * <!-- src_embed com.azure.security.keyvault.SecretClient.getSecret#string -->
+ * <pre>
+ * KeyVaultSecret secret = secretClient.getSecret&#40;&quot;secretName&quot;&#41;;
+ * System.out.printf&#40;&quot;Secret is returned with name %s and value %s%n&quot;,
+ *     secret.getName&#40;&#41;, secret.getValue&#40;&#41;&#41;;
+ * </pre>
+ * <!-- end com.azure.security.keyvault.SecretClient.getSecret#string -->
+ *
+ * <p><strong>Asynchronous Code Sample:</strong></p>
+ * <p>The following code sample demonstrates how to synchronously retrieve a previously stored secret from the Azure
+ * KeyVault, using the {@link com.azure.security.keyvault.secrets.SecretAsyncClient}.</p>
+ *
+ * <!-- src_embed com.azure.keyvault.secrets.SecretClient.getSecret#string -->
+ * <pre>
+ * secretAsyncClient.getSecret&#40;&quot;secretName&quot;&#41;
+ *     .subscribe&#40;secretWithVersion -&gt;
+ *         System.out.printf&#40;&quot;Secret is returned with name %s and value %s %n&quot;,
+ *             secretWithVersion.getName&#40;&#41;, secretWithVersion.getValue&#40;&#41;&#41;&#41;;
+ * </pre>
+ * <!-- end com.azure.keyvault.secrets.SecretClient.getSecret#string -->
+ *
+ * <hr/>
```

Comment:
Add Samples for Updating, Deleting and list secrets as above, to complete champion scenarios.